### PR TITLE
[zh] Resync the configmap page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh/docs/concepts/configuration/configmap.md [hunk 1]
```diff
@@ -381,6 +387,14 @@ ConfigMaps consumed as environment variables are not updated automatically and r
 以环境变量方式使用的 ConfigMap 数据不会被自动更新。
 更新这些数据需要重新启动 Pod。
 
+{{< note >}}
+<!--
+A container using a ConfigMap as a [subPath](/docs/concepts/storage/volumes#using-subpath) volume mount will not receive ConfigMap updates.
+-->
+将 ConfigMap 作为 [subPath](/docs/concepts/storage/volumes#using-subpath)
+卷挂载的容器无法收到 ConfiaMap 更新。
```
Comment (sftim) [inline content/zh/docs/concepts/configuration/configmap.md:395]: nit:
```suggestion
卷挂载的容器无法收到 ConfigMap 更新。
```

File: content/zh/docs/concepts/configuration/configmap.md [hunk 2]
```diff
@@ -449,6 +462,6 @@ to the deleted ConfigMap, it is recommended to recreate these pods.
   separating code from configuration.
 -->
 * 阅读 [Secret](/zh/docs/concepts/configuration/secret/)。
-* 阅读 [配置 Pod 来使用 ConfigMap](/zh/docs/tasks/configure-pod-container/configure-pod-configmap/)。
+* 阅读[配置 Pod 使用 ConfigMap](/zh/docs/tasks/configure-pod-container/configure-pod-configmap/)。
 * 阅读 [Twelve-Factor 应用](https://12factor.net/) 来了解将代码和配置分开的动机。
```
Comment (sftim) [inline content/zh/docs/concepts/configuration/configmap.md:466]: Could link to https://12factor.net/zh_cn/ if you want to.

**Review by sftim — COMMENTED** [review]
Small nits

**Review by chenxuc — CHANGES_REQUESTED** [review]
Thanks for the update. I have some minor comments.

File: content/zh/docs/concepts/configuration/configmap.md [hunk 3]
```diff
@@ -449,6 +462,6 @@ to the deleted ConfigMap, it is recommended to recreate these pods.
   separating code from configuration.
 -->
 * 阅读 [Secret](/zh/docs/concepts/configuration/secret/)。
-* 阅读 [配置 Pod 来使用 ConfigMap](/zh/docs/tasks/configure-pod-container/configure-pod-configmap/)。
-* 阅读 [Twelve-Factor 应用](https://12factor.net/) 来了解将代码和配置分开的动机。
+* 阅读[配置 Pod 使用 ConfigMap](/zh/docs/tasks/configure-pod-container/configure-pod-configmap/)。
+* 阅读 [Twelve-Factor 应用](https://12factor.net/zh/)来了解将代码和配置分开的动机。
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/concepts/configuration/configmap.md:466]: fixed.

**Review by tanjunchen — COMMENTED** [review]
/approve
/lgtm
thanks

**Pull request merged**